### ISSUE TITLE
Tailwind support/integration?

### ISSUE THREAD

**Issue by nandorojo** [nandorojo/dripsy#24]
I'm learning more about Tailwind and am curious how it might fit into the future of RN styling. I wonder if it could play a role with Dripsy somehow. I'm still getting to know it better.

Some interesting projects:

https://github.com/OwenMelbz/babel-plugin-tailwind-rn

**Comment by cmaycumber** [comment]
Interesting. I've seen a bit about tailwind but never used it personally.

I know the basic principles don't match up 1-1 to theme-ui, do you see the basic vales being used within the sx prop, a complete switch to a className based approach or working side by side somehow?

The theme-ui tailwind theme is also pretty interesting. [https://theme-ui.com/packages/tailwind/](https://theme-ui.com/packages/tailwind/)

**Comment by nandorojo** [comment]
It seems like the only crossover might be in using the same theme for both interchangeably. I'm going to try tailwind out and see if there are other ways they could work together. 

Also: https://github.com/dvkndn/typed.tw

**Comment by nandorojo** [comment]
This library by @tvke looks interesting:

Just an idea, I wonder if we could use it like so:

```jsx
import { t } from 'react-native-tailwindcss'

export default () => (
  <View tailwind={[t.mT2, { ...t.mT3, t.mB3 }]} />
)
```

Where each item in the `tailwind` array represents the styles for that breakpoint, rather than using the `md:` syntax typically used by tailwind CSS.

Some people really like atomic design patterns, and since theme io already has a tailwind adapter, could be interesting.

**Comment by saulamsal** [comment]
Any updates regarding this?

**Comment by nandorojo** [comment]
Yes actually. @axeldelafosse added a guide: https://axeldelafosse.com/blog/universal-design-system

We could add a tailwind section to the docs
